pkg/rsstorage/integration_test/integration.go

```diff
@@ -0,0 +1,63 @@
+package integrationtest
+
+// Copyright (C) 2022 by RStudio, PBC
+
+import (
+	"fmt"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+
+	"github.com/rstudio/platform-lib/pkg/rsstorage"
+	"github.com/rstudio/platform-lib/pkg/rsstorage/servers/file"
+	"github.com/rstudio/platform-lib/pkg/rsstorage/servers/postgres"
+	"github.com/rstudio/platform-lib/pkg/rsstorage/servers/s3server"
+)
+
+// Simply wraps getStorageServerAttempt, fatally erring if something goes wrong
+func GetStorageServer(cfg *rsstorage.Config, class string, destination string, waiter rsstorage.ChunkWaiter, notifier rsstorage.ChunkNotifier, pool *pgxpool.Pool, cstore rsstorage.PersistentStorageStore, debugLogger rsstorage.DebugLogger) (rsstorage.PersistentStorageServer, error) {
```

Comment:
These two functions were previously public factory methods that helped to set up storage in RSPM. However, they're not ideal since they reference all the storage server implementations and hence require importing all implementations into your application. Moving them to the `integration_test` package since they're still useful for tests.